git-hub

```diff
@@ -1513,8 +1519,12 @@ class PullCmd (IssueCmd):
                     force=args.force_push)
             infof("Creating pull request from branch {} to {}:{}",
                     remote_head, config.upstream, base)
-            pull = req.post(cls.url(), head=gh_head, base=base,
-                    title=title, body=body)
+            # The draft feature is only available in a custom media type
+            media_type = None
```

Comment:
Why not just default `media_type` to ` "application/vnd.github.v3+json"` here? 
Rather than passing down `None` all the way and finally setting it to that value in L439 above?
Basically: set the `media_type` as soon as you know what it needs to be.